packages/components/modules/messages/web/ChatRoom/ChatRoomHeader/ChatRoomOptions/index.tsx

```diff
@@ -11,11 +11,17 @@ const ChatRoomOptions: FC<ChatRoomOptionsProps> = ({
   onArchiveClicked,
   onDetailsClicked,
   onLeaveClicked,
+  onContactDetailsClicked,
 }) => (
   <MenuList>
     <MenuItem onClick={onArchiveClicked} disabled={isArchiveMutationInFlight}>
       <Typography variant="body2">{isArchived ? 'Unarchive Chat' : 'Archive Chat'}</Typography>
     </MenuItem>
+    {!isGroup && (
```

Comment:
We are checking `isGroup` twice here, the first checks if it is not a group, and the second is a ternary that returns null if it is not a group.

So maybe instead of checking this twice, we could just paste the not group part in the second part of the ternary below.